README.md

```diff
@@ -1 +1,99 @@
+| ovirt_engine_setup_answer_file_path   | UNDEF                 | Path to custom answerfile for `engine-setup`. |
+| ovirt_engine_setup_organization       | UNDEF                 | Organization name for certificate. |
+| ovirt_engine_setup_firewall_manager   | firewalld             | Specify the type of firewall manager to configure on Engine host, following values are availableL: `firewalld`,`iptables` or empty value to skip firewall configuration. |
+| ovirt_engine_setup_update_setup_packages | False              | If `True`, setup packages will be updated before `engine-setup` will be executed. Makes sense if Engine is already installed. |
+| ovirt_engine_setup_update_all_packages | True                 | If `True`, all packages will be updated before `engine-setup` will be executed. |
+| ovirt_engine_setup_accept_defaults    | False                 | If `True` use option `--accept-defaults` when executing `engine-setup`.
+
+* Engine Database:
+
+| Name                            | Default value         |  Description                                              |
+|---------------------------------|-----------------------|-----------------------------------------------------------|
+| ovirt_engine_setup_db_host            | localhost             | IP address or host name of a PostgreSQL server for Engine database. By default the database will be configured on the same host as Engine. |
+| ovirt_engine_setup_db_port            | 5432                  | Engine database port. |
+| ovirt_engine_setup_db_name            | engine                | Engine database name. |
+| ovirt_engine_setup_db_user            | engine                | Engine database user. |
+| ovirt_engine_setup_db_password        | UNDEF                 | Engine database password. |
+
+* Engine Data Warehouse Database:
+
+| Name                            | Default value         |  Description                                              |
+|---------------------------------|-----------------------|-----------------------------------------------------------|
+| ovirt_engine_setup_dwh_db_host        | localhost             | IP address or host name of a PostgreSQL server for DWH database. By default the DWH database will be configured on the same host as Engine. |
+| ovirt_engine_setup_dwh_db_port        | 5432                  | DWH database port. |
+| ovirt_engine_setup_dwh_db_name        | ovirt_engine_history  | DWH database name. |
+| ovirt_engine_setup_dwh_db_user        | ovirt_engine_history  | DWH database user. |
+| ovirt_engine_setup_dwh_db_password    | UNDEF                 | DWH database password. |
+
+* OVN related options:
+
+| Name                            | Default value         |  Description                                              |
+|---------------------------------|-----------------------|-----------------------------------------------------------|
```

Comment:
How about option to enable/disable OVN configuration during engine-setup?

Reply:
Done